posthog/__init__.py

```diff
@@ -251,6 +251,50 @@ def alias(
     )
 
 
+def capture_exception(
+    exception=None,  # type: Optional[BaseException]
```

Comment:
I think this should be mandatory, or if called without, the method is a NoOp.
If the goal is to still capture a synthetic error (there's no error instance), we should have a let's say `capture_message` which has a more semantical name

Reply:
Hmm, explained below:

> - `exception` to specify the exception to capture. If not provided, the current exception is captured via `sys.exc_info()`

Makes sense?

Although indeed I'm not sure if I'm handling the case where exc_info() is empty, will add handling for that 👍 , and some tests, since this doesn't depend on the django integration

Reply:
yep, makes a lot of sense, It's a nice fallback.